api/docs/release.dox

```diff
@@ -216,6 +216,8 @@ Further non-compatibility-affecting changes include:
    read from exact registers.
  - Added the function reg_is_vector_simd() to test whether registers are
    either XMM, YMM or ZMM, excluding any MMX register checks.
+ - Added DR_NUM_SIMD_VECTOR_REGS defintion to easily get the number of supported
```

Comment:
We can easily get the number already. Could you re-phrase into sth. like adding alias for static number of simd vector registers, and add to the commit message that proc_num_simd_registers() should be used instead whenever possible.

Reply:
Done.